### PR TITLE
Adds words to list

### PR DESCRIPTION
**Pull request by zacharysarette**
Adds a few words I came across. Will try to add more as I come across them.

# Words Added :monkey_face: 
- insert
- copy
- cut
- paste
- dependency
- resolve

Cheers!

### CODE REVIEW

**Comment by ronkorving** [comment]
Thanks for the contributions! I'll review as well as I can, probably ask one or two questions.

File: ReadMe.md [hunk 1]
```diff
@@ -122,6 +126,8 @@ Implementation | 実装 | じっそう |
 Design | 設計 | せっけい | Of code, infrastructure and systems, etc.
 Font | フォント | |
 Execution | 実行 | じっこう | Of a function or program
+Dependency | 依存関係 | いぞんかんけい
+To resolve | 解決 (する)| かいけつ | Eg. a dependency
```
Comment (ronkorving) [inline ReadMe.md:130]: I figured 解決 was really about a problem becoming resolved. Do Japanese engineers use it like this?

File: ReadMe.md [hunk 2]
```diff
@@ -82,6 +83,9 @@ Screenshot | スクショ | | Informal abbreviation
 To take a screenshot | スクリーンショットを撮る | スクリーンショットをとる |
 Video | 動画 | どうが |
 To display | 表示（する） | ひょうじ |
+To copy | コピー(する) |
+To cut | 切り取る | きりとる
+To paste | 貼り付け(する) | はりつけ
```
Comment (ronkorving) [inline ReadMe.md:88]: I also see カット＆ペースト in katakana on [jisho.org](https://jisho.org/word/%E3%82%AB%E3%83%83%E3%83%88%EF%BC%86%E3%83%9A%E3%83%BC%E3%82%B9%E3%83%88).
I am most interested in whatever is the colloquial form. Do you happen to know which that is?

Comment (zacharysarette) [reply ReadMe.md:88]: Yeah, that's fine too. カットペースト seems to also work.
If you do a wider search, it seems fine.
[DuckDuckGoSearch](https://duckduckgo.com/?q=%E3%82%AB%E3%83%83%E3%83%88%E3%83%9A%E3%83%BC%E3%82%B9%E3%83%88&t=ffab&atb=v220-2__&ia=web)

File: ReadMe.md (hunk 1 above)
Comment (sozuuuuu) [reply ReadMe.md:130]: Yes. We use `解決` as a verb.

Comment (zacharysarette) [reply ReadMe.md:130]: That's what it says on my terminal when installing a program.

File: ReadMe.md (hunk 2 above)
Comment (sozuuuuu) [reply ReadMe.md:88]: Either is OK. I'll use both depending on the context.

File: ReadMe.md (hunk 1 above)
Comment (zacharysarette) [reply ReadMe.md:130]:
```
依存関係が解決しました。
```

**Comment by ronkorving** [comment]
Thanks @zacharysarette and @sozuuuuu ^^

File: ReadMe.md (hunk 2 above)
Comment (zacharysarette) [reply ReadMe.md:88]: [wiki](https://ja.wikipedia.org/wiki/%E3%82%B3%E3%83%94%E3%83%BC%E3%83%BB%E3%82%A2%E3%83%B3%E3%83%89%E3%83%BB%E3%83%9A%E3%83%BC%E3%82%B9%E3%83%88)
According to wikipedia, カットペースト is analogous to コピペ

**Pull request merged**